graylog2-server/src/main/resources/org/graylog2/featureflag/feature-flag.config

```diff
@@ -66,4 +66,4 @@ cloud_inputs=on
 preflight_web=off
 
 # Instant archiving is disabled by default
-instant_archiving=off
+instant_archiving=on
```

Comment:
did this one slip into this PR?

Reply:
Good catch - no wonder IT-tests were failing.